src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogConnectionFormDataProvider.php

```diff
@@ -53,13 +61,11 @@ public function getData(?PunchoutCatalogConnectionTransfer $punchoutCatalogConne
          */
         return (new PunchoutCatalogConnectionTransfer())
             ->setIsActive(true)
-            ->setType('NOT_IMPLEMENTED')
-            ->setFormat('NOT_IMPLEMENTED')
-            ->setUsername('NOT_IMPLEMENTED');
+            ->setType('NOT_IMPLEMENTED');
```

Comment:
Add a `TODO` annotation to not forget to replace that value.

Reply:
It's already there, added in previous PR